### PR TITLE
Add support for "repeat click" sensitivity for number spinner

### PR DESCRIPTION
**Pull request by mattheyan**

### CODE REVIEW

File: packages/input-number/src/input-number.vue [hunk 1]
```diff
@@ -93,6 +93,10 @@
         default: -Infinity
       },
       value: {},
+      sensitivity: {
+        type: Number,
+        default: 100
+      },
```
Comment (mattheyan) [inline packages/input-number/src/input-number.vue:99]: Added `sensitivity` prop to allow the "scrolling" behavior to be configured to be faster or slower as needed.

File: packages/input-number/src/input-number.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
       role="button"
       :title="t('el.spinner.decrease')"
       v-if="controls"
-      v-repeat-click="decrease"
+      v-repeat-click:[sensitivity]="decrease"
```
Comment (mattheyan) [inline packages/input-number/src/input-number.vue:16]: This syntax (a dynamic "argument" to the directive) is advertised in the Vue.js documentation: https://vuejs.org/v2/guide/custom-directive.html#Dynamic-Directive-Arguments

File: src/directives/repeat-click.js [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+    }
+  }
+  return duration;
+}
+
 export default {
   bind(el, binding, vnode) {
     let interval = null;
     let startTime;
+    let duration = getRepeatDuration(binding.arg, vnode, 100);
```
Comment (mattheyan) [inline src/directives/repeat-click.js:24]: Make the repeat interval dynamic based on the directive argument (with 100 used as the default).

File: src/directives/repeat-click.js [hunk 4]
```diff
@@ -1,12 +1,30 @@
 import { once, on } from 'element-ui/src/utils/dom';
 
+function getRepeatDuration(bindingArg, vnode, defaultDuration) {
+  let duration = defaultDuration;
+  if (bindingArg) {
+    let arg = bindingArg;
+    const dynamicArgMatch = typeof arg === 'string' ? /^\[(.*)\]$/.exec(arg) : null;
```
Comment (mattheyan) [inline src/directives/repeat-click.js:7]: I found that the dynamic argument approach in the Vue.js documentation didn't work automatically, but could be made do work using the `vnode` argument (already used for the function/expression below).
Also, here's a SO post explaining the approach: https://stackoverflow.com/questions/45223018/how-to-pass-dynamic-argument-in-vuejs-custom-directive/48631316#48631316
And, here's a post where Evan You announces it in Vue 2.6: https://medium.com/the-vue-point/vue-2-6-released-66aa6c8e785e#ca69

File: packages/input-number/src/input-number.vue (hunk 2 above)
Comment (tskimmett) [reply packages/input-number/src/input-number.vue:16]: Cool. I was not aware of this feature.

**Review by tskimmett — APPROVED** [review]

**Pull request merged**